### PR TITLE
feat(alerts): Add new onboarding illustration

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Looks like this

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+const AlertsImage = styled('img')`
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    user-select: none;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 220px;
+    margin-top: auto;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx:34]:
```suggestion
    margin: auto 0;
```

File: src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import {t} from 'app/locale';
+import ButtonBar from 'app/components/buttonBar';
+import OnboardingPanel from 'app/components/onboardingPanel';
+import emptyStateImg from 'app/../images/spot/alerts-empty-state.svg';
+
+type Props = {
+  actions: React.ReactNode;
+};
+
+function Onboarding({actions}: Props) {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx:13]: Please, why did you decide not to use func component?

File: src/sentry/static/sentry/app/views/alerts/list/index.tsx [hunk 3]
```diff
@@ -162,31 +163,23 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
     navigateTo(`/settings/${params.orgId}/projects/:projectId/alerts/`, router);
   };
 
-  tryRenderFirstVisit() {
+  tryRenderOnboarding() {
     const {firstVisitShown} = this.state;
 
     if (!firstVisitShown) {
-      return null;
+      //return null;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/alerts/list/index.tsx:170]: will you need this? wouldn't it be better to remove?

**Review by priscilawebdev — COMMENTED** [review]
I love this new illustration 😍

File: src/sentry/static/sentry/app/views/alerts/list/index.tsx (hunk 3 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/alerts/list/index.tsx:170]: yes I will :)

File: src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx:13]: Like `React.FC` type?

**Review by ckj — APPROVED** [review]
😍

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/alerts/list/onboarding.tsx:13]: yes

**Pull request merged**